Makefile

```
@@ -27,9 +27,9 @@ build: export SPDK_GIT_BRANCH != git -C spdk name-rev --name-only HEAD
 build: export SPDK_GIT_COMMIT != git rev-parse HEAD:spdk
 build: export BUILD_DATE != date -u +"%Y-%m-%dT%H:%M:%SZ"
 
-up: SVC = nvmeof ## Services
-up: OPTS ?= --abort-on-container-exit
-up: override OPTS += --no-build --remove-orphans --scale nvmeof=$(SCALE)
+up: ## Launch services
+up: SVC ?= ceph nvmeof ## Services
+up: OPTS ?= --abort-on-container-exit --exit-code-from $(SVC) --remove-orphans --scale nvmeof=$(SCALE)
```

Now looking at this, I would probably split that into 2, because we probably don't want to loss the `$(SCALE)` option:
```suggestion
up: OPTS ?= --abort-on-container-exit --exit-code-from $(SVC) --remove-orphans
up: override OPTS += --scale nvmeof=$(SCALE)
```